portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,71 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto+Mono:300,400");
+@import url("https://fonts.googleapis.com/css?family=Josefin+Sans:300,400,700");
+html {
+  padding: 0px;
+  margin: 0px;
+}
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  width: 100vw;
+  height: 100vh;
+  padding: 0;
+  margin: 0;
+  font-family: "Roboto Mono", monospace, sans-serif;
+  font-size: 16px;
+  color: #28313d;
+  background: #fff;
+  overflow-x: hidden;
+  scroll-behavior: smooth;
 }
 
-#greeting-container {
-  margin-top: 20px;
+*::before,
+*::after {
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
```

Comment:
Add a blank line between the css rules through the file.

Reply:
I added a blank line between all css rules. Please, let me know if this looks ok. Thank you